packages/web/src/components/gcds-icon/gcds-icon.tsx

```diff
@@ -36,19 +40,25 @@ export class GcdsIcon {
   /**
    * Defines the size of the icon.
    */
-  @Prop() size?: 'inherit' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' = 'inherit';
+  @Prop() padding?: 'none' | 'fixed-width' = 'none';
```

Comment:
Since this prop doesn't actually add any padding, but calculates the width instead, how about renaming this prop to `fixedWidth` and making it a boolean? If it's true, you can add the `fixed-width` class that calculates the fixed with, otherwise, the class won't be added.

Reply:
yeah, lets do that!